src/main/java/tracko/model/item/Quantity.java

```diff
@@ -8,8 +8,8 @@
  */
 public class Quantity {
     public static final String MESSAGE_CONSTRAINTS = "Quantity should be valid."
-            + "\nA valid Quantity value should be numeric, non-empty, "
-            + "non-negative, and should not exceed 2,147,483,647.";
+            + "\nA valid Quantity value should be an integral, non-empty, "
```

Comment:
I think for this, it's either `should be integral` or `should be an integer`, bc integral is an adjective :D

Reply:
I think here, integral works fine because it is describing the `value` (noun) below

Reply:
ah i completely missed the 'value' in the line after! sorry about that